sql/moz-fx-data-shared-prod/google_ads_derived/conversion_event_categorization_v1/query.sql

```diff
@@ -36,15 +44,9 @@ clients_last_seen_raw AS (
   JOIN
     clients_first_seen_14_days_ago clients
     ON cls.client_id = clients.client_id
-  WHERE
-    cls.submission_date >= '2023-11-01' --first cohort date
-    AND cls.submission_date
-    BETWEEN cls.first_seen_date
-    AND DATE_ADD(cls.first_seen_date, INTERVAL 6 DAY) --get first 7 days from their first main ping
-    --to process less data, we only check for pings between @submission date - 15 days and submission date + 15 days for each date this runs
     AND cls.submission_date
-    BETWEEN DATE_SUB(@report_date, INTERVAL 1 DAY) --15 days before DS
-    AND DATE_ADD(@report_date, INTERVAL 29 DAY) --15 days after DS
+    BETWEEN clients.first_main_ping
```

Comment:
```suggestion
    -- join the clients_last_seen so that we get the first 7 days of each client's main ping records (for the clients that sent > 0 main pings in their first week)
    BETWEEN clients.first_main_ping
```